code/game/gamemodes/gang/gang.dm

```diff
@@ -193,6 +193,8 @@ var/list/gang_colors_pool = list("red","orange","yellow","green","blue","purple"
 	var/removed
 
 	for(var/datum/gang/G in gangs)
+		if(G.fighting_style == "multiverse" && !remove_bosses)
```

Comment:
or give ```/datum/gang``` a ```is_deconvertible``` var, defaulting to true that just so happens to be false on the multiverse one. cmon @KorPhaeron think generic here.